packages/components/package.json

```diff
@@ -48,14 +48,15 @@
     "@ember/string": "^3.1.1",
     "@ember/test-waiters": "^3.1.0",
     "@embroider/addon-shim": "^1.9.0",
+    "@embroider/macros": "^1.16.10",
     "@floating-ui/dom": "^1.6.12",
-    "@hashicorp/design-system-tokens": "^2.3.0",
-    "@hashicorp/flight-icons": "^3.9.0",
+    "@hashicorp/design-system-tokens": "workspace:*",
+    "@hashicorp/flight-icons": "workspace:*",
```

Comment:
While this could speed up development, it is best practice to specify a fixed version for dependencies in packages that are intended to be consumed by others (more predictable and stable behavior).

More importantly, many of our consumers (boundary, nomad, vault) benefit from these two packages via `@hashicorp/design-system-components` – operating this change will likely mean they will not be able to resolve paths to our tokens and icons.

Reply:
reverted